packages/ResizeObserver/src/ResizeObserver.js

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import PropTypes from "prop-types";
+import useResizeObserver from "use-resize-observer/polyfilled";
+import debounce from "lodash.debounce";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import * as sc from "./ResizeObserver.styles";
+
+const propTypes = {
+  breakpointLarge: PropTypes.number,
```

Comment:
Is it obvious enough that these values refer to widths?

Reply:
Hmm, I think some explanation will be required to know how the breakpoints work no matter what.  But we might want height breakpoints in the future, and then might need to introduce a breaking change to the API for distinguishing – although I think there's a good chance we will never need that either.  `width` is a far more common use case, and I'd like to keep it simple where possible.  `breakpointLargeWidth` and `breakpointSmallWidth` are kinda ugly, but they ARE more clear... so I am conflicted about this 🤔